api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPrepared.md

```diff
@@ -53,6 +53,6 @@ The editor's value.
 The editor's width.
 
```

Comment:
Can you update line 47 above?

Line 47:
`Gets and sets the delay between when a user stops typing the field's value and when it is applied.`

I'd like to make "it" more specific. Can we say the following?

`Gets and sets the delay between when a user stops typing the field's value and when the change is applied.`

Reply:
Noted

Reply:
Thanks.